### DISCUSSION TITLE
Sainsmart using ESP3D for belt printer

### DISCUSSION THREAD

**Discussion by rudetrooper** [luc-github/ESP3D#674] [General]
https://youtu.be/BPqRx_RJcqA Sainsmart is claiming to have octoprint built in to the printer but the interface looks like ESP3D.

**Comment by luc-github** [comment]
haha it  seems they know what they sell https://www.sainsmart.com/collections/new-arrivals/products/sainsmart-infi-20-belt-3d-printer-infinite-z
ESP3D is visible in window ... wrong claim.... 
It show Marlin on ESP32 Board using ESP3DLib and ESP3D webUI 
![image](https://user-images.githubusercontent.com/8822552/134632579-6281cd3a-f834-4602-8a74-4a5ccb456a04.png)


thanks for sharing
 ;)

**Reply by rudetrooper** [reply]
You should leave a comment on the video, it would be funny. Also contact them directly and educate the marketing department. Foosel(Octoprint) is probably going to contact once she has time.

**Reply by luc-github** [reply]
there is already a comment under video so I did one directly on sainsmart website, but not sure it is working....

**Reply by rudetrooper** [reply]
Foosel also [tweeted about it using the official Octoprint account](https://twitter.com/OctoPrint3D/status/1441456319362441218). I'm not surprised that the contact us page doesn't work. Unfortunately twitter might be your best chance to get in contact with SainSmart.

**Reply by luc-github** [reply]
thank you, I got also the information on discord about this tweet, I do not use twitter,
I hope  octoprint will get clear answer, it is misleading for users

**Reply by luc-github** [reply]
I though it was just wrong screen used, but it seems octoprint has not been reach out.. .

On other hand esp3d was used as claim by several companies without being contacted, so I am really surprised by the situation.

I am lookiing forward to know the conclusion

**Reply by rudetrooper** [reply]
Foosel is not happy about the situation. Its not even difficult for these companies to reach out and ask you guys for permission.

**Reply by luc-github** [reply]
yes I can understand that, it is why i put this note in read me:
`If you use ESP3D on your product, drop me a message so I can link your product page here.`
to stimulate the  communication but....

**Reply by luc-github** [reply]
I have added a comment under youtube video https://youtu.be/BPqRx_RJcqA to express ESP3D feedback also

**Reply by rudetrooper** [reply]
They responded to Foosel after she sent an email to marketing@sainsmart.com with an apology and an explanation. They've also updated the listing on their website, apparently they added an ESP chip so ESP3D-WEBUI is included by default. 
![Screenshot_20210926-135037_Chrome](https://user-images.githubusercontent.com/189071/134818734-aef6d545-487d-46b7-a356-48a7e9a09202.jpg)
I don't think the marketing department properly understand how octoprint works. They're missing the steps for setting up octoprint and setting up the API key for octoremote. Users will probably have to supply their own raspberry pi and setup octoprint.

**Reply by luc-github** [reply]
wow, thank you for sharing - how did you got so fast and precise information ? 

So their main board is actually an ESP32 ? I am curious to know what based board is it - with or without IO expander, etc...

Yes some marketing are really not serious...
